client_local_api/api_test_utils.go

```diff
@@ -0,0 +1,30 @@
+package client_local_api
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+//TODO move this to random port for testing
+const serverBindAddress = ":8000"
+
+func startServer() {
+	go Bootstrap(serverBindAddress)
+}
+
+func doGet(t *testing.T, path string) *http.Response {
+
+	resp, err := http.Get("http://localhost" + serverBindAddress + "/" + path)
+	assert.Nil(t, err)
+	return resp
+}
+
+func expectJsonAndStatus(t *testing.T, resp *http.Response, httpStatus int, v interface{}) {
```

Comment:
I would have 2 functions for different responsibilities:
- `expectJsonAndStatus(t, response, 200)`
- `expectJsonBody(t, response, `{"status":"ok"}`)` <-- maybe `httptest` already has JSON assertions

Reply:
I had such helper:
```
func assertResponseJson(t *testing.T, response *httptest.ResponseRecorder, expectedJsonFile string) {
	expectedResponse, err := filesystem_helper.ReadFile(expectedJsonFile)
	if err != nil {
		t.Fatalf("Error reading file: %s. %s", expectedJsonFile, err.Error())
	}

	assert.JSONEq(t, string(expectedResponse), response.Body.String())
}
```